### PR TITLE
Civ 6012 Ga ccd  changes for the new camunda flow for HearingScheduled Process

### PR DESCRIPTION
**Pull request by deepthidoppalapudihmcts**
**Before creating a pull request make sure that:**

- [x ] commit messages are meaningful and follow good commit message guidelines
- [x ] README and other documentation has been updated / added (if needed)
- [x ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-6012

### Change description ###

GA CCD changes for the new camunda flow for Hearing Scheuled GA Event  - GA CaseProgression

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: ga-ccd-definition/FixedLists/HearingChannel-GAR3CP-nonprod.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "ID": "HearingChannel",
```
Comment (karthick-mohanasundaram-HMCTS) [inline ga-ccd-definition/FixedLists/HearingChannel-GAR3CP-nonprod.json:3]: Could we reuse the existing one?

File: ga-ccd-definition/FixedLists/HearingLengthCasePro-GAR3CP-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "ID": "HearingLengthCasePro",
```
Comment (karthick-mohanasundaram-HMCTS) [inline ga-ccd-definition/FixedLists/HearingLengthCasePro-GAR3CP-nonprod.json:3]: could we reuse the existing one?

Comment (deepthidoppalapudihmcts) [reply ga-ccd-definition/FixedLists/HearingLengthCasePro-GAR3CP-nonprod.json:3]: Created a new JIRA task for refactoring https://tools.hmcts.net/jira/browse/CIV-6823

File: ga-ccd-definition/FixedLists/HearingChannel-GAR3CP-nonprod.json (hunk 1 above)
Comment (deepthidoppalapudihmcts) [reply ga-ccd-definition/FixedLists/HearingChannel-GAR3CP-nonprod.json:3]: Created a new JIRA task for refactoring https://tools.hmcts.net/jira/browse/CIV-6823

**Review by pliao-hmcts — APPROVED** [review]

**Pull request merged**